KsApi/models/UpdateTests.swift

```diff
@@ -70,4 +70,40 @@ internal final class UpdateTests: XCTestCase {
       update.value?.urls.web.update
     )
   }
+
+  func testJSONDecoding_WithBadUrlsWebData() {
```

Comment:
I think since you added two new methods here you can use a name like `testJSONDecoding_WithBadUrls_WebData` and the one below can be `testJSONDecoding_WithBadUrls_WrongKeyData`